src/components/addDiceModal.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import './addDiceModal.css'
+
+const faces = [
+  4,
+  6,
+  10,
+  20,
+]
+
+const AddDiceModal = ({ addDice }) => {
```

Comment:
maybe you could try lifecycle hooks to give back the focus to the opening button when the modal is closed
_optional_

Reply:
#11